ui/tests/acceptance/clients/counts/overview-test.js

```diff
@@ -63,11 +68,14 @@ module('Acceptance | clients | overview', function (hooks) {
     assert.dom(CHARTS.xAxisLabel).exists({ count: 7 }, 'chart months matches query');
   });
 
-  // TODO revisit once CE changes are finalized
-  test.skip('it should update charts when querying date ranges', async function (assert) {
+  test('it should update charts when querying date ranges', async function (assert) {
     // query for single, historical month with no new counts (July 2023)
+    const service = this.owner.lookup('service:version');
+    service.type = 'community';
+
     const licenseStartMonth = format(LICENSE_START, 'yyyy-MM');
     const upgradeMonth = format(UPGRADE_DATE, 'yyyy-MM');
+    const endMonth = format(STATIC_PREVIOUS_MONTH, 'yyyy-MM');
```

Comment:
Just an FYI there's a `subMonths` [function that exists ](https://date-fns.org/v4.1.0/docs/subMonths) as well